### ISSUE TITLE
Phinch visualization in "16S Microbial Analysis with mothur" tutorial 

### ISSUE THREAD

**Issue by Jalalalzanin** [galaxyproject/training-material#2183]
Dear all
I have problem with the visualization the  composition of microbial community with Phinch, the output of the tool "Make.biom" as shown below 
![image](https://user-images.githubusercontent.com/46455885/101276649-6eac5100-37bf-11eb-8b19-f979137f6457.png)
I tried many times to repeat the step but the output failure, metadata file was downloaded again to the history to confirm there is no problem with earlier downloading, however the result is the same 
![image](https://user-images.githubusercontent.com/46455885/101276725-f1351080-37bf-11eb-98e4-21ccd343ec30.png)

thank you in advance

**Comment by yvanlebras** [comment]
Dear @Jalalalzanin thank you for reporting this issue. Can you look at the standard error message if available, from an error dataset and/or can you maybe share your history so we can investigate in details? 
Thank you

**Comment by Jalalalzanin** [comment]
Dear yvanlebras thank you for response 
no error not message 
https://usegalaxy.eu/u/jalal_alznin85/h/16s-microbial-analysis-with-mothur-extended

**Comment by Jalalalzanin** [comment]
Dear yvanlebras thank you for response
no error message wad detected
https://usegalaxy.eu/u/jalal_alznin85/h/16s-microbial-analysis-with-mothur-extended

**Comment by Jalalalzanin** [comment]
Dear yvanlebras thank you for response
no error message was detected
https://usegalaxy.eu/u/jalal_alznin85/h/16s-microbial-analysis-with-mothur-extended

**Comment by yvanlebras** [comment]
Thank you. I can't look at the datasets as I think you have to unshared the history then reshare it specifying you want to share also all elements of the history (checking the corresponding box). Can you try and re paste here the link if the history (will normally be the same but just to be sure)

**Comment by Jalalalzanin** [comment]
that's clear @yvanlebras 
https://usegalaxy.eu/u/jalal_alznin85/h/16s-microbial-analysis-with-mothur-extended 
thank you

**Comment by yvanlebras** [comment]
Ok, will look at it ASAP. ITM, here you can't have access to stderr because you "only" have a data collection as output, but if you check the box ~"output log file: yes" so you can then look at the error on the failed output log file through the "bug" button

**Comment by yvanlebras** [comment]
The error message is not interesting here unfortunately. But looking at the standard output, here is a message who can help:

'ERROR]: can't find metadata information for F3D0_, quitting.'

 But not sure I submit the same job... But trying without metadata file as input seem ok. When specifying the metadata file I have this aforementioned message.

Here are my rapid tests: https://usegalaxy.eu/u/ylebras/h/imported-16s-microbial-analysis-with-mothur-extended

**Comment by Jalalalzanin** [comment]
Thank you @yvanlebras 
That's correct I tried without metadata file. 
![image](https://user-images.githubusercontent.com/46455885/101283227-2f472a00-37ea-11eb-9c09-aeb73e6fd4ca.png)
and here the result with Phinch
![image](https://user-images.githubusercontent.com/46455885/101283281-687f9a00-37ea-11eb-97b3-3b4d4ef28b51.png)

**Comment by yvanlebras** [comment]
Yes, but After that, if you select "taxonomy bar chart" phinch say that the metadata file is required :( so you have to look into details to find the error with the metadata file and maybe more with the filtered otu input file...

**Comment by Jalalalzanin** [comment]
that's right @yvanlebras 
I will check the metadata file and repeat this step
I appreciate you help
